docs/source/developing/index.adoc

```diff
@@ -0,0 +1,14 @@
+[[developing-index]]
+= Developing Minishift
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+This section contains information about how to extend, build, and release Minishift, and
+how to contribute to various aspects of the project.
+
+- link:../developing/developing.html[Developing Minishift]
```

Comment:
I don't get it. You are own the developing directory. The link is just './developing.html'. You are effectively going up one directory and then down again.